flexget/plugins/search_sceneaccess.py

```diff
@@ -234,6 +232,11 @@ def search(self, task, entry, config=None):
             Search for entries on SceneAccess
         """
 
+        session = task.requests
+
+        if not 'sceneaccess.eu' in session.domain_limiters:
```

Comment:
pep standard prefers `if string not in bla bla` and I'm a firm believer in adhering to standards *hint hint*